core/app/models/spree/credit_card.rb

```diff
@@ -95,6 +95,10 @@ def cc_type=(type)
       end
     end
 
+    def brand=(type)
+      cc_type = type
```

Comment:
I'm thinking this should be `self.cc_type =` otherwise looks like a variable assignment to me 🤔

Reply:
I turned it into an `alias_method`